.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Feature request
+about: Proposal for a new feature or enhancement
+---
+
+Please review our [Contribution guide](https://github.com/vmware/clarity/blob/master/CONTRIBUTING.md).
+
+## Summary
+
+Describe the change or feature in detail. Try to answer the following questions.
+
+* What is the change?
+* What is a use case for this change?
+* Why should it go in Clarity?
+* Does this change impact existing behaviors? If so how?
+* If this change introduces a new behavior, is this behavior accessible?
+
+## Examples
+
+_If possible, show examples of the change. You may create one yourself, or link to external sites that have the idea. It can also be very useful to prototype the idea in isolation outside of Clarity with a [Stackblitz example](https://stackblitz.com/@clr-team)._
+
+## API
+
+_Describe the intended API for the feature you want to add. This would include:_
+
+* CSS classes and DOM structure for pure static UI contribution, and inputs/outputs, components, directives, services, or anything that is exported publicly for Angular contributions.
+* Examples of code snippets using this new feature.
+* Note very clearly if anything **might** be a breaking change.
+
+_In the case of bug fixes or internal changes, there will most likely be no API changes._
+
+## Implementation Plan
```

Comment:
People can request feature without filling in a full proposal template. What you created here is not `.github/ISSUE_TEMPLATE/feature_request.md`, it's `.github/ISSUE_TEMPLATE/contribution_proposal.md`. This is not going to work for 90% of the feature requests we receive.

Please keep the feature request template to something minimal like:
1. Desired feature
2. Examples and use cases
3. Current workaround if it exists